portfolio/src/main/webapp/style.css

```diff
@@ -38,19 +42,22 @@ body, html {
   font-size: 4rem;
 }
 
-.school-title {
+.title {
   display: block;
   text-align: center;
-  color: white;
   padding-top: 4rem;
   font-size: 3.5rem;
 }
 
+.title.school {
+  color: white;
+}
+
 #music, #research {
   display: none;
 }
 
-.school-div {
+.page-div {
```

Comment:
This is nor following alphabetic order?

Reply:
Just want to confirm that: after I give approval with one comment, Github won't let you merge the PR until you resolve the comment, right?

Reply:
Github technically lets me merge with approval without resolving the comment, I think - at least in this case, I have the ability to merge. I'm not sure what the official rules are.
Re: alphabetization, I was planning on alphabetizing all the CSS after these PRs were merged because I was worried that moving all the attributes around might lead to some conflicts. Is this reasonable?

Reply:
Sure, it is reasonable. Usually in Google we left a comment here:

//TODO(LDAP): Alphabetize the CSS files.